### PR TITLE
Bugfixes in client panel

### PR DESCRIPTION
**Pull request by sowmiyasubramaniyan**
Fix: #1973 & #1977 
Bugs are elaborated in the below document with the module index as the client.
https://docs.google.com/spreadsheets/d/1CB139UvzvUtB1JOHYP0nmEqjxdoHQPGWLGjD_niH9pw/edit#gid=1379931937

### CODE REVIEW

File: app/Http/Controllers/Common/MailChimpController.php [hunk 1]
```diff
@@ -105,7 +105,7 @@ public function addSubscriberByClientPanel(Request $request)
             if ($exe['status'] == 400) {
                 $error = $exe['detail'];
 
-                return errorResponse($error, 400);
+                return errorResponse('Member already exists!', 400);
```
Comment (RafficMohammed) [inline app/Http/Controllers/Common/MailChimpController.php:108]: use lang

File: app/Http/Controllers/Front/PageController.php [hunk 2]
```diff
@@ -108,14 +108,15 @@ public function edit($id)
     public function store(Request $request)
     {
         $this->validate($request, [
-            'name' => 'required',
+            'name' => 'required|unique:frontend_pages,name',
```
Comment (RafficMohammed) [inline app/Http/Controllers/Front/PageController.php:111]: use validation file for this

**Review by RafficMohammed — CHANGES_REQUESTED** [review]
use trans everywhere and test cases too

File: app/Http/Controllers/Front/PageController.php [hunk 3]
```diff
@@ -127,9 +121,13 @@ public function store(Request $request)
             $this->page->parent_page_id = $request->input('parent_page_id');
             $this->page->type = $request->input('type');
             $this->page->content = $request->input('content');
-            $this->page->save();
+            if ($pages_count <= 2) {
```
Comment (sandesh556) [inline app/Http/Controllers/Front/PageController.php:124]: You don't need to use if else for return conditions

File: public/js/intl/js/utils.js [hunk 4]
```diff
@@ -358,7 +358,7 @@ PE:[,[,,"(?:[14-8]|9\\d)\\d{7}",,,,,,,[8,9],[6,7]],[,,"(?:(?:4[34]|5[14])[0-8]\\
 "$1 $2 $3",["2[14-68]|3[26-9]|4[1246-8]|6(?:1|75)|7[1-35]|8[1-36]"],"(0$1)"],[,"(\\d{2})(\\d{3})(\\d{4})","$1 $2 $3",["87"]],[,"(\\d{3})(\\d{6})","$1 $2",["9(?:[5-79]|8[1-6])"],"0$1"],[,"(\\d{3})(\\d{3})(\\d{3})","$1 $2 $3",["[2-8]"],"0$1"],[,"(\\d{4})(\\d{3})(\\d{4})","$1 $2 $3",["9"]]],,[,,,,,,,,,[-1]],,,[,,,,,,,,,[-1]],[,,"[2-9]0\\d{4,7}",,,,"201234567",,,[6,7,8,9]],,,[,,,,,,,,,[-1]]],QA:[,[,,"[2-7]\\d{7}|(?:2\\d\\d|800)\\d{4}",,,,,,,[7,8]],[,,"(?:23|4[04])\\d{6}",,,,"44123456",,,[8]],[,,"(?:28|[35-7]\\d)\\d{6}",
```
Comment (sandesh556) [inline public/js/intl/js/utils.js:358]: why is this here?

**Review by sandesh556 — CHANGES_REQUESTED** [review]

File: public/js/intl/js/utils.js (hunk 4 above)
Comment (sowmiyasubramaniyan) [reply public/js/intl/js/utils.js:358]: I have made a change in this file

**Review by sandesh556 — DISMISSED** [review]

**Review by RafficMohammed — CHANGES_REQUESTED** [review]
Write test cases for the request validation field too.

**Review by sandesh556 — APPROVED** [review]

**Review by SarfrazS — CHANGES_REQUESTED** [review]
Below Issues are not fixed Please fix and update me.
 - [ ] Mismatch in displaying the alert message and field name showing on the page.
- [ ] The calendar option is not proper, Not able to select the particular year in the calendar on the Edit page.
- [ ] 2FA Alert message is showing wrong when we are passing invalid.

**Review by SarfrazS — CHANGES_REQUESTED** [review]
Increase the number of values in name. as of not taking only 10.
image
Edit/Update Publishing Date not showing correct. i updated from 13/02/2023 to  14/92/2023 but showing as below.
image

**Review by SarfrazS — APPROVED** [review]
Pointed issue fixed now
image

**Pull request merged**